src/plotman/monitor.py

```diff
@@ -0,0 +1,243 @@
+                plots_layout.update(jobs_table)
+
+                live.refresh()
+                for _ in range(10):
+                    keys = prompt_toolkit_input.read_keys()
+                    quit_keys = {'q', prompt_toolkit.keys.Keys.ControlC}
+                    if any(key.key in quit_keys for key in keys):
+                        return
+                    time.sleep(0.1)
+
+
+async def with_prompt_toolkit():
+    config_path = plotman.configuration.get_path()
+    config_text = plotman.configuration.read_configuration_text(config_path)
+    cfg = plotman.configuration.get_validated_configs(config_text, config_path)
+
+    tmp_prefix = os.path.commonpath(cfg.directories.tmp)
+    dst_prefix = os.path.commonpath(cfg.directories.dst)
+
+    plots_buffer = prompt_toolkit.layout.controls.Buffer()
+    disks_buffer = prompt_toolkit.layout.controls.FormattedTextControl()
+    rows = [
+        header_window,
+        plots_window,
+        disks_window,
+        archive_window,
+        logs_window,
+    ] = [
+        prompt_toolkit.layout.containers.Window(),
+        prompt_toolkit.layout.containers.Window(
+            prompt_toolkit.layout.controls.BufferControl(buffer=plots_buffer),
+        ),
+        prompt_toolkit.layout.containers.Window(content=disks_buffer),
+        prompt_toolkit.layout.containers.Window(),
+        prompt_toolkit.layout.containers.Window(),
+    ]
+
+    root_container = prompt_toolkit.layout.containers.HSplit(rows)
+
+    layout = prompt_toolkit.layout.Layout(root_container)
+
+    key_bindings = prompt_toolkit.key_binding.KeyBindings()
+
+    key_bindings.add('q')(exit_key_binding)
+
+    application = prompt_toolkit.Application(
+        layout=layout,
+        full_screen=True,
+        key_bindings=key_bindings,
+    )
+
+    rich_console = rich.console.Console(
+        # file=io.StringIO,
+        force_terminal=True,
+        # color_system="truecolor",
+    )
+
+    jobs = []
+
+    async with anyio.move_on_after(10):
+        async with anyio.create_task_group() as task_group:
+            task_group.start_soon(functools.partial(
+                cancel_after_application,
+                application=application,
+                cancel_scope=task_group.cancel_scope,
+            ))
+
+            for i in itertools.count():
+                disks_buffer.text = str(i)
+
+                jobs = plotman.job.Job.get_running_jobs(
+                    cfg.directories.log,
+                    cached_jobs=jobs,
+                )
+                jobs_data = build_jobs_data(
+                    jobs=jobs,
+                    dst_prefix=dst_prefix,
+                    tmp_prefix=tmp_prefix,
+                )
+
+                jobs_table = build_jobs_table(jobs_data=jobs_data)
+                with rich_console.capture() as capture:
+                    rich_console.print(jobs_table)
+
+                plots_buffer.text = capture.get()
```

Comment:
I get output like this instead of black on white text.  Any ideas how to get the fancy output from `rich` into `prompt_toolkit` properly?

``^[[7m ^[[0m^[[7m#^[[0m^[[7m ^[[0m^[[7m ^[[0m^[[7mplot_id ^[[0m^[[7m ^[[0m^[[7
```